pkg/sso/client.go

```diff
@@ -198,7 +198,7 @@ func (c *Client) GetProfile(ctx context.Context, opts GetProfileOptions) (Profil
 // PromoteDraftConnectionOptions contains the options to pass in order to
 // promote a draft connection.
 type PromoteDraftConnectionOptions struct {
-	ID string `json:"id"`
+	Token string `json:"id"`
```

Comment:
Should we rename the key as well?

Reply:
I actually know how this works. @maxchehab ?